SEED-platform/seed
# Increase AnalysisMessage.user_message length

#### Any background context you want to provide?
In the BETTER Analysis workflow, any error messages retrieved from a BETTER response will be saved on the AnalysisMessage model. The current database length of these messages is set to 255 and any messages longer than 255 will be truncated and a '...' placed on the end. Once this happens there is no way to retrieve the whole message. 

#### What's this PR do?
Increases the length of `user_messages `and `debug_messages` from 255 to 1024.

#### How should this be manually tested?
Create a BETTER analysis when a property has no `property-type` set. Check the analysis for the full error message.

#### What are the relevant tickets?
#3891 
#### Screenshots (if appropriate)
![Screenshot 2023-03-24 at 3 59 36 PM](https://user-images.githubusercontent.com/58446472/227651338-6dbc7307-3a87-40c9-980f-cecfe9b2573a.png)
